.database_generator/mangos_translation/lightshope_merge.py

```
@@ -0,0 +1,201 @@
+      converted = _clean_sql_content(infile.read())
+  finally:
+    try:
+      os.remove(temp_path)
+    except OSError:
+      pass
+
+  target_conn.executescript(converted)
+
+
+def _ensure_row_exists(conn: sqlite3.Connection, table: str, key_col: str, key_val):
+  conn.execute(f"INSERT OR IGNORE INTO {table} ({key_col}) VALUES (?)", (key_val,))
```

Ruff (0.14.6)

Possible SQL injection vector through string-based query construction

(S608)